lv_conf_template.h

```diff
@@ -1314,6 +1314,22 @@
 /** Benchmark your system */
 #define LV_USE_DEMO_BENCHMARK 0
 
+#if LV_USE_DEMO_BENCHMARK
+    /** Option to use static aligned fonts */
+    #define LV_DEMO_BENCHMARK_ALIGNED_FONTS 0
+    
+    #if LV_DEMO_BENCHMARK_ALIGNED_FONTS
+        /** Configure which static fonts to be used for benchmark */
+        #define LV_FONT_BENCHMARK_MONTSERRAT_12_ALIGNED 0
+        #define LV_FONT_BENCHMARK_MONTSERRAT_14_ALIGNED 1
+        #define LV_FONT_BENCHMARK_MONTSERRAT_16_ALIGNED 1
+        #define LV_FONT_BENCHMARK_MONTSERRAT_18_ALIGNED 0
+        #define LV_FONT_BENCHMARK_MONTSERRAT_20_ALIGNED 1
+        #define LV_FONT_BENCHMARK_MONTSERRAT_24_ALIGNED 1
+        #define LV_FONT_BENCHMARK_MONTSERRAT_26_ALIGNED 1
+    #endif
```

Comment:
```suggestion
   /*Use fonts where bitmaps are aligned 16 byte and has Nx16 byte stride*/
    #if LV_DEMO_BENCHMARK_ALIGNED_FONTS   1
```

Reply:
> I think we go for either all aligned or all unaligned. Is there a good use case for enabling them one by one?

This was the most configurable way. If you want to go with the option to have all of them enabled or disabled, I can update it accordingly

Reply:
Okay, please have a single config then. We need to find a  balance between flexibility and the length of `lv_conf.h`. :smile: